1_assignment/alphbet_frequency.js

```diff
@@ -1,55 +1,32 @@
-
-
-//Sample String .
+// Sample String .
 var string = "u@#d$fkj$#!kds";
 
-
-
-// Solution 1 : Using String Functions.
-function calculateFrequencyUsingString(string) {
-  // Write your code here
-  var i=0;
-  frequency=new Array(26).fill(0);
-  for (i = 0; i < string.length ; i++)
-  {
-  	if (string.charCodeAt(i) >= 97 && string.charCodeAt(i) <= 122)
-  	{
-  		frequency[(string.charCodeAt(i)-97)]++;
+// Solution 1: Using String Functions.
+function calculateFrequencyUsingString(string){
+  var frequency = new Array(26).fill(0);
+  for (var i = 0; i < string.length; i++){
+  	if (string.charCodeAt(i) >= 97 && string.charCodeAt(i) <= 122){
+  		frequency[(string.charCodeAt(i) - 97)]++;
   	}
   }
-  
   return frequency;
 }
 
-
-
-// Solution 2  : using only Arrays and not using String functions.
-
-function calculateFrequencyUsingArrays(string) {
-  // Write your code here
-  var i=0;
-  var j=0;
-  var frequency=new Array(26).fill(0);
-  var alphabet_array=["a" , "b" ,"c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"];
-  for (i=0 ; i < string.length ; i++)
-  {
-  	for (j = 0; j < 26 ; j++)
-  	{
-      {
-  			if(alphabet_array[j] == string[i])
-  			{
+// Solution 2: using only Arrays and not using String functions.
+function calculateFrequencyUsingArrays(string){
+  var frequency = new Array(26).fill(0);
+  var alphabet_array = ["a" , "b" ,"c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"];
```

Comment:
I am not fan of this logic, why do I need an array of all the alphabets to match against? The problem statement requires frequency of only the alphabets occurring in the given string.